aar_doc/templates/markdown.j2

```diff
@@ -29,10 +29,10 @@ Tags: {{ metadata.galaxy_info.galaxy_tags | join(', ') }}
 {{ argument_specs[entrypoint].description }}
 {% else %}
 {%- for line in argument_specs[entrypoint].description -%}
-{{ line }}
-{% endfor -%}
-{% endif -%}
-{% endif -%}
+{{ line }}  
+{% endfor %}
```

Comment:
the removed `-` is generating some empty lines where they are not wanted, hence the failing tests.

Would you be up to generate a testcase for the described scenario so that we can work on fixing this without affecting our other test cases?

Reply:
Can do - however I currently have two different implementations for this part: The one with linebreaks in one paragraph (here) and the one with each line as its own paragraph (#127, which seems to not break existing tests). I'm not sure which one I like more, but I'm somewhat favoring the latter because I think the two spaces at the end of a line are rather ugly and prone to be removed by accident.

I'm therefore considering closing this PR in favor of #127, and add test cases for multiline descriptions as well as markup rendering to the other PR.

@schurzi what are your thoughts on this?

Reply:
I agree, let's close this and continue in #127